tests/test-amp-style-sanitizer.php

```diff
@@ -130,11 +166,19 @@ public function get_body_style_attribute_data() {
 	 * @param string $source               Source.
 	 * @param string $expected_content     Expected content.
 	 * @param string $expected_stylesheets Expected stylesheets.
+	 * @param array  $expected_errors      Expected error codes.
 	 */
-	public function test_body_style_attribute_sanitizer( $source, $expected_content, $expected_stylesheets ) {
+	public function test_body_style_attribute_sanitizer( $source, $expected_content, $expected_stylesheets, $expected_errors = array() ) {
```

Comment:
Maybe the 3rd `$expected_errors` parameter isn't necessary, given that the `dataProvider` [get_link_and_style_test_data](https://github.com/Automattic/amp-wp/pull/1048/files#diff-78b16003565f81870be4068700bdbfc5R200) always has a 3rd array item of `array()`. But this isn't a blocker to approving.

Reply:
Yeah, you're right. But I suppose it aligns with the other tests and opens the door to testing for validation errors in the future.